hello_world_on_chain/package.json

```diff
@@ -9,7 +9,6 @@
   "license": "MIT",
   "dependencies": {
     "@nomicfoundation/hardhat-toolbox": "^5.0.0",
-    "@openzeppelin/contracts": "^5.2.0",
```

Comment:
since we simplified, we no longer need the user contract to be ownable and so we don't need openzeppelin